packages/plugin-core/src/commands/MoveHeader.ts

```diff
@@ -32,16 +35,25 @@ import { VSCodeUtils } from "../utils";
 import { BasicCommand } from "./base";
 import _ from "lodash";
 import path from "path";
-import { getEngine } from "../workspace";
+import { getDWorkspace, getEngine, getVaultFromUri } from "../workspace";
 import { EngineAPIService } from "../services/EngineAPIService";
 import { delayedUpdateDecorations } from "../features/windowDecorations";
 import { findReferences, FoundRefT } from "../utils/md";
 import { Location } from "vscode";
 import { file2Note, vault2Path } from "@dendronhq/common-server";
+import {
+  NewLocation,
+  PickerUtilsV2,
+  ProviderAcceptHooks,
+} from "../components/lookup/utils";
 
 type CommandInput =
   | {
       dest?: NoteProps;
+      initialValue?: string;
```

Comment:
why do we have both `dest` and `initialValue`? aren't they the same thing?

Reply:
if we have to pick one, i would use `initialValue` to keep compatibility with `NoteLookup`

Reply:
you are right. dest is a remnant of the old version of move header that I used for easier testing. will get rid of it